x11-misc/xkbset/xkbset-0.8-r4.ebuild

```
@@ -0,0 +1,41 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
```

Always use the latest available EAPI, 8 should work here.
```suggestion
EAPI=8
```